Index: testPostDisplay.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><title>Test Post</title>\n<?php include \"template.php\";\n/**  @var $conn */\n?>\n<!--Pulls the details frome the Posts table-->\n<?php\n$postDetails = $conn->query( \"SELECT BodyText, Title FROM Posts\");\n?>\n\n<!--this will be the border of the hole post-->\n<div>\n\n<?php\nwhile ($postData = $postDetails->fetchArray()){\n?>\n\n\n<!--    this is the div that will display the title and other things displayed in the heaad note-->\n    <div>\n<?php echo '<h1>' . $postData[1] . '</h1>'; ?>\n    </div>\n<!--    this is the div that will display the contents of the body of the post-->\n    <div>\n<?php echo $postData[0]; ?>\n    </div>\n<!--    this it the div that will display the contents of the fotter of the post eg. the up-votes and down-votes-->\n    <div>\n\n    </div>\n    <?php\n}\n    ?>\n</div>\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/testPostDisplay.php b/testPostDisplay.php
--- a/testPostDisplay.php	(revision 22ac26dd556732e59a0daca3ade13164a7104f05)
+++ b/testPostDisplay.php	(date 1692331915504)
@@ -1,19 +1,20 @@
 <title>Test Post</title>
+<link rel="stylesheet" href="style.css">
 <?php include "template.php";
 /**  @var $conn */
 ?>
+
 <!--Pulls the details frome the Posts table-->
 <?php
 $postDetails = $conn->query( "SELECT BodyText, Title FROM Posts");
 ?>
 
-<!--this will be the border of the hole post-->
-<div>
 
 <?php
 while ($postData = $postDetails->fetchArray()){
 ?>
-
+<!--this will be the border of the hole post-->
+<div class="POST">
 
 <!--    this is the div that will display the title and other things displayed in the heaad note-->
     <div>
@@ -27,7 +28,7 @@
     <div>
 
     </div>
-    <?php
-}
-    ?>
 </div>
+<?php
+}
+?>
\ No newline at end of file
Index: style.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/style.css b/style.css
new file mode 100644
--- /dev/null	(date 1692331915608)
+++ b/style.css	(date 1692331915608)
@@ -0,0 +1,12 @@
+.POST{
+    border-radius: 25px;
+height: 500px;
+}
+
+.POSTTITLE{
+    background-color: #5c636a;
+}
+
+.POSTBODY{
+    background-color: #2b3035;
+}
\ No newline at end of file
