Index: testPostDisplay.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><title>Test Post</title>\n<?php include \"template.php\";\n/**  @var $conn */\n?>\n<!--Pulls the details frome the Posts table-->\n<?php\n$postDetails = $conn->query( \"SELECT BodyText, Title FROM Posts\");\n?>\n\n<!--this will be the border of the hole post-->\n<div>\n\n<?php\nwhile ($postData = $postDetails->fetchArray()){\n?>\n\n\n<!--    this is the div that will display the title and other things displayed in the heaad note-->\n    <div>\n<?php echo '<h1>' . $postData[1] . '</h1>'; ?>\n    </div>\n<!--    this is the div that will display the contents of the body of the post-->\n    <div>\n<?php echo $postData[0]; ?>\n    </div>\n<!--    this it the div that will display the contents of the fotter of the post eg. the up-votes and down-votes-->\n    <div>\n\n    </div>\n    <?php\n}\n    ?>\n</div>\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/testPostDisplay.php b/testPostDisplay.php
--- a/testPostDisplay.php	(revision 76a9ecebba716d1f47cedef14a2b213851bf97c2)
+++ b/testPostDisplay.php	(date 1692332712684)
@@ -1,21 +1,27 @@
-<title>Test Post</title>
+<?php include "template.php";
+/**  @var $conn */
+?>
+    <title>Test Post</title>
+<link rel="stylesheet" href="style.css">
+<body>
+    <h1>5</h1>
 <?php include "template.php";
 /**  @var $conn */
 ?>
-<!--Pulls the details frome the Posts table-->
+
+<!--Pulls the details from the Posts table-->
 <?php
 $postDetails = $conn->query( "SELECT BodyText, Title FROM Posts");
 ?>
 
-<!--this will be the border of the hole post-->
-<div>
 
 <?php
 while ($postData = $postDetails->fetchArray()){
 ?>
+<!--this will be the border of the hole post-->
+<div class="POST">
 
-
-<!--    this is the div that will display the title and other things displayed in the heaad note-->
+<!--    this is the div that will display the title and other things displayed in the head note-->
     <div>
 <?php echo '<h1>' . $postData[1] . '</h1>'; ?>
     </div>
@@ -27,7 +33,11 @@
     <div>
 
     </div>
-    <?php
+</div>
+    <p>1</p>
+
+    </body>
+
+<?php
 }
-    ?>
-</div>
+?>
\ No newline at end of file
Index: style.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/style.css b/style.css
new file mode 100644
--- /dev/null	(date 1692332078112)
+++ b/style.css	(date 1692332078112)
@@ -0,0 +1,12 @@
+.POST{
+    border-radius: 25px;
+height: 500px;
+}
+
+.POSTTITLE{
+    background-color: #5c636a;
+}
+
+.POSTBODY{
+    background-color: #2b3035;
+}
\ No newline at end of file
